<launch>

  <!-- Launch the grid map filters demo node. -->
  <node pkg="gpg_elevation_mapping" type="grid_map_filters" name="grid_map_filters" output="screen">
    <!-- Input topic -->
  	<param name="input_topic" value="/elevation_mapping/elevation_map_raw" />
  	<!-- Output topic -->
  	<param name="output_topic" value="/grid_map_filter/filtered_map" />
  	<!-- Load grid map filter chain configuration -->
    <rosparam command="load" file="$(find grid_map_demos)/config/filters_demo_filter_chain.yaml" />
  </node>

</launch>
